docs/shared/angular-standalone-tutorial/angular-standalone.md

```diff
@@ -0,0 +1,795 @@
+   │        │  │     └─ ui.component.ts
+   │        └─ ...
+   ├─ ...
+   ├─ src
+   │  ├─ app
+   │  │  ├─ ...
+   │  │  ├─ app.component.ts
+   │  ├─ ...
+   ├─ ...
+```
+
+Each of these libraries
+
+- has its own `project.json` file with corresponding targets you can run (e.g. running tests for just orders: `nx test modules-orders`)
+- Nx names these projects based on the `--directory` flag, e.g. `modules-orders`; you can find the name in the corresponding `project.json`
+- has a dedicated `index.ts` file which is the "public API" of the library
+- is mapped in the `tsconfig.base.json` at the root of the workspace
+
+### Importing Libraries into the Angular Application
+
+All libraries that we generated are automatically mapped over the root-level `tsconfig.base.json`.
+
+```json {% fileName="tsconfig.base.json" %}
+{
+  "compilerOptions": {
+    ...
+    "paths": {
+      "@myngapp/modules/products": ["modules/products/src/index.ts"],
+      "@myngapp/modules/orders": ["modules/orders/src/index.ts"],
+      "@myngapp/modules/shared/ui": ["modules/shared/ui/src/index.ts"]
+    },
+    ...
+  },
+}
+```
+
+Hence we can easily import them into other libraries and our Angular application. For example: let's use our existing `ProductsComponent` in `modules/products/src/lib/products/`:
+
+```ts
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'myngapp-products',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './products.component.html',
+  styleUrls: ['./products.component.css'],
+})
+export class ProductsComponent {}
+```
+
+Make sure the `ProductsComponent` is exported via the `index.ts` file of our `products` library (which it should already be). This is our public API with the rest of the workspace. Only export what's really necessary to be usable outside the library itself.
+
+```ts {% fileName="modules/products/src/index.ts" %}
+export * from './lib/products/products.component';
+```
+
+We're ready to import it into our main application now. If you opted into generating a router configuration when setting up the Nx workspace initially, you should have a `app.routes.ts` file in your `app` folder. If not, create it and configure the Angular router.
+
+Configure the routing as follows:
+
+```ts {% fileName="src/app/app.routes.ts" %}
+import { Route } from '@angular/router';
+import { NxWelcomeComponent } from './nx-welcome.component';
+
+export const appRoutes: Route[] = [
+  {
+    path: '',
+    component: NxWelcomeComponent,
+    pathMatch: 'full',
+  },
+  {
+    path: 'products',
+    loadComponent: () =>
+      import('@myngapp/modules/products').then((m) => m.ProductsComponent),
+  },
+];
+```
+
+As part of this step, we should also remove the `NxWelcomeComponent` from the `AppComponent.imports` declaration s.t. it is just loaded over the routing mechanism. The `app.component.html` should just have the `<router-outlet>` left:
```

Comment:
```suggestion
As part of this step, we should also remove the `NxWelcomeComponent` from the `AppComponent.imports` declaration so that it is just loaded over the routing mechanism. The `app.component.html` should just have the `<router-outlet>` left:
```